server/src/main/java/io/crate/types/DataType.java

```diff
@@ -266,6 +266,13 @@ public final StorageSupport<? super T> storageSupportSafe() {
         return storageSupport;
     }
 
+    /**
+     * @return {@code true} if values should always be loaded from stored fields
+     */
+    public boolean retrieveFromStoredFields() {
```

Comment:
I'd say this also belongs to `StorageSupport` because if there's no storage support this info is kinda irrelevant.